snow/consensus/snowball/binary_slush.go

```diff
@@ -7,6 +7,12 @@ import "fmt"
 
 var _ BinarySlush = (*binarySlush)(nil)
 
+func newBinarySlush(choice int) binarySlush {
+	return binarySlush{
```

Comment:
I feel like we dont need this function and we can use the struct directly

Reply:
I don't feel super strongly but it fits with the pattern elsewhere so 🤷